viewer/packages/cad-geometry-loaders/src/sector/sectorUtilities.ts

```diff
@@ -129,3 +129,23 @@ export function findSectorMetadata(root: SectorMetadata, sectorId: number): Sect
   }
   return foundSector;
 }
+
+/**
+ * Like THREE.Box3.containsBox(), but with fuzziness added.
```

Comment:
Can you explain a little more here what is going on?

Reply:
Added explanation and improved function name